jellyfin/jellyfin-androidtv
# Include Jellyfin Exoplayer FFmpeg dep

**Changes**
Port the includes from the Android repo as-is into this repo to include
the dependency

Convert our initialisation to prefer the built in extensions. This
should avoid flaky device logic and matches the Android app choice too.

**Issues**
Partially Completes: #1308 

**To Test**
- Grab a video with DTS audio (free trailers with DTS can be downloaded from: https://thedigitaltheater.com/dts-trailers/)
- Ensure the playback is set to Exoplayer 
- Ensure playback is direct (not downsampling)
- Play media, ensure audio now plays

This should hopefully fix some of the issues people (including myself) are seeing with certain audio formats
